devops/lib/config.rb

```diff
@@ -0,0 +1,11 @@
+def application
+  'EDx'
+end
+
+def environment
+  'Stage'
+end
+
+def elasticsearch_domain
+  [application, environment].join('-').downcase
```

Comment:
Looks like we could go ahead and set up our definitions of `application` and `environment` to look for environment variables `||` the strings we're currently defining them as.